client/feature/ProfilePhotoAndName.js

```diff
@@ -21,7 +22,8 @@ export const ProfilePhotoAndNameFeature = {
             edit: EditPhotoAndName,
             checkForErrors: checkPhotoAndNameAsync,
             makePreview: previewPhotoAndName,
-        }]
+        }],
+        profilePhotoLayers: [VerificationBadge]
```

Comment:
I think this should be a different feature.

ProfilePhotoAndName is a feature applied to profiles, that makes the profile names and photos editable.

The feature you are writing is applied to conversations, and shows a badge on people's photos. 
These should probably be different features.

We currently don't have a feature list what structures they apply to, since that would get awkward features that add functionality to things like bylines that apply to multiple structures, but it's possible it would make sense to have some kind of convention for documenting what we think a feature is supposed to be applied to.